docs/storage.md

```diff
@@ -1,4 +1,94 @@
-# Storage (CSI)
+# Storage
+
+## Bundled OpenEBS storage
+
+K0s comes out with bundled OpenEBS installation which can be enabled by using [configuration file](./configuration.md)
+
+Use following configuration as an example:
+
+```yaml
+spec:
+  extensions:
+    storage:
+      type: openebs_local_storage
+```
+
+The cluster wil have two storage classes available for you to use:
+
+```shell
+bash-5.1# k0s kc get sc
```

Comment:
The long format would be easier to understand in the example:
`k0s kubectl get storageclass`